### DISCUSSION TITLE
I can't add my plugins to Craft Plugins Catalog

### DISCUSSION THREAD

**Discussion by Awilum** [craftcms/.github#17] [Craft Console] [answered]
I don't see any form or button to do this in my Craft Console Panel.

Also in doc I see 

```
From your Craft Console account, you’ll need to first go to “Account Settings”, make sure “Enable plugin developer features” is checked under your username, and choose “Save”.
```

But I don't see this setting in my profile and I can't add plugins to craft plugins catalog.

how to fix this?

**Comment by brandonkelly** [comment, marked as answer]
Apologies, those docs need to be updated. You must create an Organization within Craft Console first. Only Organizations can publish plugins in the Plugin Store now.
